kidash/kidash.py

```diff
@@ -1588,45 +1636,52 @@ def fetch_dashboard(elastic_url, dash_id, es_index=None):
                     {"id": index_pattern_id,
                      "value": get_index_pattern_json(elastic,
                                                      index_pattern_id)}
-                    )
+                )
 
     return kibana
 
 
 def export_dashboard_files(dash_json, export_file, split_index_patterns=False):
-
     if os.path.isfile(export_file):
         logging.info("%s exists. Remove it before running.", export_file)
-        raise RuntimeError("%s exists. Remove it before running." % export_file)
+        raise RuntimeError("%s exists. Remove it before running."
+                           % export_file)
 
     with open(export_file, 'w') as f:
         if not split_index_patterns:
             f.write(json.dumps(dash_json, indent=4, sort_keys=True))
         else:
             index_patterns = dash_json.pop("index_patterns")
 
-            with open(export_file, 'w') as f:
-                f.write(json.dumps(dash_json, indent=4, sort_keys=True))
+            with open(export_file, 'w') as fa:
+                fa.write(json.dumps(dash_json, indent=4, sort_keys=True))
```

Comment:
since `f` is already in use, changing this file object to `fa`

Reply:
Good!